packages/docs/src/routes/docs/(qwikcity)/action/index.mdx

```diff
@@ -122,6 +122,69 @@ export default component$(() => {
 
 In the example above, the `addUser` action is triggered when the user clicks the button. The `action.submit()` method returns a `Promise` that resolves when the action is done.
 
+## Actions with Event Handlers
+
+We can use the `onSubmitCompleted$` event handler after an action is executed successfully and returns some data. It is particularly useful when you need to perform additional tasks, such as resetting UI elements or updating the application state, once an action has been completed successfully.
+
+Here's an example of the `onSubmitCompleted$` handler used to edit an item in a EditForm component of a todo app.
```

Comment:
I think the example has a bit too much out of scope code (like styles and such) as well as missing context of the parent passing down the signal.

AFK right now, but ideally the example would be the bare minimum code.

I'll see if I can get around to showing what I mean soon.

Reply:
yeah it might make sense to make a super simple version then link to an example thats more complex

Reply:
Given that there is no docs at all about this, i am happy to merge and iterate, at least we have something

Reply:
> Given that there is no docs at all about this, i am happy to merge and iterate, at least we have something

Should I edit and remove the styles? Along with taking out the Icons and replacing X with "Cancel" for example?